sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql

```diff
@@ -87,43 +90,52 @@ SELECT
   first_seen_date = submission_date
   AND reported_first_session_ping AS is_new_install,
   first_seen_date = submission_date AS is_new_profile,
+  COALESCE(activations.activated, FALSE) AS is_activated,
+  -- Metrics / Facts
+  IF(COALESCE(new_activations.activated, FALSE), 1, 0) AS activations_count,
+  IF(COALESCE(client_days.client_id IS NOT NULL, FALSE), 1, 0) AS client_count,
```

Comment:
Quick update on what's happening here. The `activations_count` includes clients who were not necessarily active on this day; which means we now need to include a `client_count` to get the actual DAU (but maybe `client_count` is the wrong name -`active_client_count`? `DAU`? Open to suggestions)